lib/libc/stdlib/malloc/mrs/mrs.c

```diff
@@ -705,9 +705,7 @@ quarantine_should_flush(struct mrs_quarantine *quarantine, bool is_free)
 static inline uint64_t
 cheri_revoke_get_cyc(void)
 {
-#if defined(__mips__)
-	return (cheri_get_cyclecount());
-#elif defined(__riscv)
+#if defined(__riscv)
 	return (__builtin_readcyclecounter());
```

Comment:
This isn't really a smart thing to be using for RISC-V anyway. The cycle counter is per-hart, and we're not pinning threads in userspace as far as I know. This should probably be reading the time CSR instead, which is global, and more akin to x86's TSC and Arm's CNTVCT.